listenbrainz/listenstore/timescale_listenstore.py

```diff
@@ -1005,11 +1009,15 @@ def delete_listen(self, listened_at: int, user_name: str, recording_msid: str):
                       AND user_name = :user_name
                       AND data -> 'track_metadata' -> 'additional_info' ->> 'recording_msid' = :recording_msid """
 
+        query_decr_lc = """ UPDATE listen_count 
```

Comment:
I think this function should do the following:
- Start a transaction
- First query for affected users who have listen counts that have not been committed yet.
- Then run a query that tallies up the listen counts for the users from above.
- Execute an update query that sets new total values for each user.
- Commit transaction
- Update high water mark.

Reply:
Uh, I think this comment got misplaced. This function is to decrement the listen count for a user when they delete one of their listens. I think you meant to comment this on the cron job to bring listen counts up to date, no?